cfme/tests/configure/test_register_appliance.py

```diff
@@ -174,6 +175,20 @@ def test_rh_registration(appliance, request, reg_method, reg_data, proxy_url, pr
     request.addfinalizer(appliance.unregister)
 
 
+def test_rhsm_registration_check_repo_names(temp_appliance_preconfig_funcscope, soft_assert):
+    """ Checks default rpm repos on a fresh appliance """
+    ver = temp_appliance_preconfig_funcscope.version.series()
+    with temp_appliance_preconfig_funcscope:
```

Comment:
**REQUIRED:** I think we should add a TODO here, **TODO:** We shouldn't have to do this. The only reason we need the context manager here is that ```RedHatUpdates``` doesn't yet support Collections.